src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example2.java

```diff
@@ -0,0 +1,13 @@
+/*xml
+<module name="Checker">
+  <module name="FileLength">
+    <property name="max" value="1500"/>
```

Comment:
Please do us a favor.
Change max to be "5" and make Example2 to be violation.

Please make example3 to show usage of `fileExtensions` property. In example3 set fileExtensions=txt
And show fact that java is skipped and violation is on txt file only .

Here is example of how we did file extension example in another Check https://github.com/checkstyle/checkstyle/commit/6c2405453f800bdb742c7e31411b57cc826a0d00#diff-e5ba5fe56a9f8ea2a93c39f4df1ddc2b1359c3828695652f98a01893b418b200R140

https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/2cbdc32_2024163715/checks/misc/newlineatendoffile.html#Example4-code

Reply:
Done